src/App.tsx

```diff
@@ -1,6 +1,20 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import React, { useState } from 'react';
```

Comment:
Consider renaming `sortItems` to something more action-oriented, such as `getSortedItems`, to better align with the checklist recommendation that function names should start with a verb.